sdk/highlight-php/src/SDK/HighlightOpenTelemetry.php

```diff
@@ -0,0 +1,151 @@
+    private static float $timeout = 30.;
+        
+
+    public function __construct(Highlight $highlight)
+    {
+        $options = $highlight->getOptions();
+
+        $attributes = [
+            HighlightAttributes::HIGHLIGHT_PROJECT_ID => $options->getProjectId(),
+            ResourceAttributes::DEPLOYMENT_ENVIRONMENT => $options->getEnvironment(),
+            ResourceAttributes::SERVICE_INSTANCE_ID => SessionId::get()->__toString(),
+            ResourceAttributes::SERVICE_VERSION => $options->getVersion(),
+            ResourceAttributes::SERVICE_NAME => $options->getServiceName()    
+        ];
+
+        if ($options->isMetricEnabled()) {
+            $attributes[ResourceAttributes::TELEMETRY_SDK_NAME] = HighlightVersion::getSdkName();
+            $attributes[ResourceAttributes::TELEMETRY_SDK_VERSION] = HighlightVersion::getSdkVersion();
+            $attributes[ResourceAttributes::TELEMETRY_SDK_LANGUAGE] = HighlightVersion::getSdkLanguage();
+            
+            $attributes[ResourceAttributes::TELEMETRY_DISTRO_NAME] = 'highlight-opentelemetry-php-sdk';
+            $attributes[ResourceAttributes::TELEMETRY_DISTRO_VERSION] = phpversion('opentelemetry');
+
+            $attributes[HighlightAttributes::TELEMETRY_PHP_VERSION] = HighlightVersion::getPhpVersion();
+            $attributes[HighlightAttributes::TELEMETRY_PHP_VENDOR] = HighlightVersion::getPhpOsName();
+            $attributes[HighlightAttributes::TELEMETRY_PHP_VERSION_DATE] = HighlightVersion::getPhpOsVersion();
+            $attributes[ResourceAttributes::OS_NAME] = HighlightVersion::getPhpOsName();
+            $attributes[ResourceAttributes::OS_VERSION] = HighlightVersion::getPhpOsVersion();
+            $attributes[ResourceAttributes::OS_TYPE] = HighlightVersion::getPhpOsArch();
+        }
+
+        foreach ($options->getDefaultAttributes() as $key => $value) {
+            $attributes[$key] = $value;
+        }
+        $attributesFactory = new AttributesFactory();
+        $attributesBuilder = $attributesFactory->builder($attributes);
+        
+
+        $resource = ResourceInfo::create($attributesBuilder->build());
+
+        // Tracer
+        $transport = (new OtlpHttpTransportFactory())
+            ->create(HighlightRoute::buildTraceRoute($options->getBackendUrl()), 
+                'application/json', 
+                self::$headers, 
+                self::$compression, 
+                self::$timeout);
+        
+        $exporter = new SpanExporter($transport);
+        $spanProcessor = new BatchSpanProcessor($exporter, ClockFactory::getDefault());
+        $sampler = new AlwaysOnSampler();
+
+        $this->tracerProvider = (new TracerProviderBuilder())
+            ->addSpanProcessor($spanProcessor)
+            ->setResource($resource)->setSampler($sampler);
+
+
+        // Log
+        $transport = (new OtlpHttpTransportFactory())->create(HighlightRoute::buildLogRoute($options->getBackendUrl()), 'application/json');
```

Comment:
can we put `application/json` into a class attribute similar to `self::$compression`?

Reply:
Sure. Added them as:
```php
    private static string $format = 'application/json';
    private static array $headers = [];
    private static string $compression = 'gzip';
    private static float $timeout = 30.;
```